genie-web/src/main/resources/static/scripts/Page.js

```diff
@@ -53,7 +53,7 @@ export default class Page extends React.Component {
 
   loadPageData(query) {
     if ($.isEmptyObject(query)) {
-      query = { size: 25 };
+      query = { size: 100 };
```

Comment:
@cabhishek I'm assuming this will apply to all pages, right? (i.e. not just jobs).

@tgianos given the lazy loading of tags we saw yesterday, maybe we should hold off on this, WDYT?

Reply:
I think the original request from @rdblue, amongst others, mostly pertained to jobs which aren't impacted by this since tags aren't included in the top level page.

However, @cabhishek is the preference saved if a user selects `100` as the load size? If so these users could just set it themselves. If not I don't necessarily see much harm leaving this at 100 for the jobs page since the search there already returns the proper projection.

Reply:
@tgianos Preference is not saved in a cookie. But the `size` param is present in the URL and persisted across different server requested e.g during pagination or during link share.

Reply:
@mprimi yes, all pages now have size of `100` by default.